test/engine/engine.test.ts

```diff
@@ -102,31 +100,25 @@ for (const fixture of fixtures) {
       return await plugin.makeCurrencyTools().then(async currencyTools => {
         tools = currencyTools
 
-        keys = await tools.createPrivateKey(WALLET_TYPE)
-        const info: EdgeWalletInfo = {
+        const privateKeys = await tools.createPrivateKey(WALLET_TYPE)
+        privateWalletInfo = {
           id: '1',
           type: WALLET_TYPE,
-          keys
+          keys: privateKeys
         }
-        const keys2 = await tools.derivePublicKey(info)
-        keys = Object.assign(keys, keys2)
+        const publicKey = await tools.derivePublicKey(privateWalletInfo)
+        Object.assign(privateWalletInfo.keys, publicKey)
       })
     })
   })
 
   describe(`Make Engine for Wallet type ${WALLET_TYPE}`, function () {
     it('Make Engine', async function () {
       if (WALLET_TYPE === 'wallet:fio') this.timeout(60000)
-      const info: EdgeWalletInfo = {
-        id: '1',
-        type: WALLET_TYPE,
-        // @ts-expect-error
-        keys
-      }
       // eslint-disable-next-line @typescript-eslint/strict-boolean-expressions
       if (!plugin) throw new Error('ErrorNoPlugin')
       return await plugin
-        .makeCurrencyEngine(info, currencyEngineOptions)
+        .makeCurrencyEngine(privateWalletInfo, currencyEngineOptions)
```

Comment:
Should we pass in a public wallet info, using the `publicKey` above, to make this a better test?

Reply:
Yes, but FIO is blocking this from passing at least for now.